modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/AbstractSortedIndexStorageTest.java

```diff
@@ -604,148 +605,148 @@ void testScanSimple() {
+                ),
                 contains(1, 2, 3)
         );
 
         // Let's check only with the lower bound.
         assertThat(
-                indexStorage.scan(
-                                serializer.serializeRowPrefix(1),
-                                null,
-                                (GREATER_OR_EQUAL | LESS_OR_EQUAL)
-                        ).stream()
-                        .map(indexRow -> serializer.deserializeColumns(indexRow)[0])
-                        .collect(toList()),
+                scan(
+                        indexStorage,
+                        serializer.serializeRowPrefix(1),
+                        null,
+                        (GREATER_OR_EQUAL | LESS_OR_EQUAL),
+                        AbstractSortedIndexStorageTest::firstArrayElement
+                ),
                 contains(1, 2, 3, 4)
         );
 
         assertThat(
-                indexStorage.scan(
-                                serializer.serializeRowPrefix(1),
-                                null,
-                                (GREATER_OR_EQUAL | LESS)
-                        ).stream()
-                        .map(indexRow -> serializer.deserializeColumns(indexRow)[0])
-                        .collect(toList()),
+                scan(
+                        indexStorage,
+                        serializer.serializeRowPrefix(1),
+                        null,
+                        (GREATER_OR_EQUAL | LESS),
+                        AbstractSortedIndexStorageTest::firstArrayElement
+                ),
                 contains(1, 2, 3, 4)
         );
 
         assertThat(
-                indexStorage.scan(
-                                serializer.serializeRowPrefix(1),
-                                null,
-                                (GREATER | LESS_OR_EQUAL)
-                        ).stream()
-                        .map(indexRow -> serializer.deserializeColumns(indexRow)[0])
-                        .collect(toList()),
+                scan(
+                        indexStorage,
+                        serializer.serializeRowPrefix(1),
+                        null,
+                        (GREATER | LESS_OR_EQUAL),
+                        AbstractSortedIndexStorageTest::firstArrayElement
+                ),
                 contains(2, 3, 4)
         );
 
         assertThat(
-                indexStorage.scan(
-                                serializer.serializeRowPrefix(1),
-                                null,
-                                (GREATER | LESS)
-                        ).stream()
-                        .map(indexRow -> serializer.deserializeColumns(indexRow)[0])
-                        .collect(toList()),
+                scan(
+                        indexStorage,
+                        serializer.serializeRowPrefix(1),
+                        null,
+                        (GREATER | LESS),
```

Comment:
Just a question, why did you add parentheses here?

Reply:
For clarity